community/stunnel/APKBUILD

```diff
@@ -49,13 +45,15 @@ package() {
 	install -Dm755 "$srcdir"/stunnel.initd "$pkgdir"/etc/init.d/stunnel
 	install -m644 "$srcdir"/stunnel.conf "$pkgdir"/etc/stunnel/stunnel.conf
 
+	rm -rf "$pkgdir"/usr/share/doc
+
```

Comment:
Why is it needed to remove the doc dir?

Reply:
My mistake. For some reason i thought it contained certificate files from the testsuite.
Reverted.